packages/builder-vite/vite-config.ts

```diff
@@ -97,10 +97,20 @@ export async function pluginConfig(options: ExtendedOptions, _type: PluginConfig
       // These are the svelte stories we need to exclude from HMR
       const storyPatterns = ['**/*.story.svelte', '**/*.stories.svelte'];
       // Non-story svelte files
-      plugins.push(sveltePlugin({ ...svelteOptions, exclude: [...userExclude, ...storyPatterns] }));
+      // Starting in 1.0.0-next.42, svelte.config.js is included by default.
+      // We disable that, but allow it to be overridden in svelteOptions
+      plugins.push(sveltePlugin({ configFile: false, ...svelteOptions, exclude: [...userExclude, ...storyPatterns] }));
```

Comment:
Does this also need to only use the first plugin if it's an array?

Reply:
No, the reason we need to for the other one is that we're expecting a single object, spreading the values, and changing the `name` field.  Vite itself is fine with arrays of plugins, it flattens them all out at the end.